product-service/utils/productValidator.js

```diff
@@ -0,0 +1,8 @@
+export default (product) => {
+  return (
+    typeof product.title === "string" &&
```

Comment:
You are not checking the values to be not empty. It means you can create a product with title = '"".

Reply:
Thanks for the comments! I will update them in the next PR.